x-pack/test/functional/page_objects/monitoring_page.js

```diff
@@ -5,11 +5,26 @@
  */
 
 export function MonitoringPageProvider({ getPageObjects, getService }) {
-  const PageObjects = getPageObjects(['common', 'header']);
+  const PageObjects = getPageObjects(['common', 'header', 'shield', 'spaceSelector']);
   const testSubjects = getService('testSubjects');
+  const security = getService('security');
 
   return new class MonitoringPage {
-    async navigateTo() {
+    async navigateTo(useSuperUser = false) {
+      // always create this because our tear down tries to delete it
+      await security.user.create('basic_monitoring_user', {
```

Comment:
Shouldn't this maybe be inside this: https://github.com/elastic/kibana/pull/47865/files#diff-b5773aa68cf0bba75970470c4b391bdeR21 if condition?

Reply:
I added the comment:
> // always create this because our tear down tries to delete it

Because we are always trying to remove it here: https://github.com/elastic/kibana/pull/47865/files#diff-328ca8d955b871223f4463b5a7ce9721R39

I guess I could `try/catch` that call and do nothing if it fails? I'm not sure the best way here, but at least this way gets the tests to pass (which was a challenge honestly)

Reply:
Ah, I see now, this makes sense